pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -1267,7 +1269,9 @@ func (qjm *XController) ScheduleNext() {
 					if qjm.quotaManager != nil {
 						var msg string
 						var preemptAWs []*arbv1.AppWrapper
-						quotaFits, preemptAWs, msg = qjm.quotaManager.Fits(qj, aggqj, proposedPreemptions)
+						//quota trees fail to update with AW deletes, release quota before assigning
+						qjm.quotaManager.Release(qj)
```

Comment:
I am not sure if this makes any sense here now, it seems superfluous. Have you observed a scenario where having this call here would be beneficial?